manifests/params_agent.pp

```diff
@@ -233,6 +236,11 @@
 
       $manage_repo = true
 
+      $default_rootcheck_system_audit = [
+        "./shared/system_audit_rcl.txt",
+        "./shared/system_audit_ssh.txt",
+      ]
+
```

Comment:
No need to use an auxiliary variable default_rootcheck_system_audit.

Changed in 624daa7 & 057f54e.

Thanks,